invenio_app_rdm/theme/assets/semantic-ui/js/invenio_app_rdm/landing_page/ManageDefaultBrandingAction/ManageDefaultBrandingAction.js

```diff
@@ -0,0 +1,119 @@
+// This file is part of InvenioRDM
+// Copyright (C) 2023 CERN.
+//
+// Invenio App RDM is free software; you can redistribute it and/or modify it
+// under the terms of the MIT License; see LICENSE file for more details.
+
+import React, { Component } from "react";
+import { Button, Label, Message, Popup, Icon } from "semantic-ui-react";
+import { i18next } from "@translations/invenio_app_rdm/i18next";
+import { http } from "react-invenio-forms";
+import PropTypes from "prop-types";
+
+export class ManageDefaultBrandingAction extends Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      loading: false,
+      error: undefined,
+    };
+  }
+
+  handleSetBranding = async (value) => {
+    const { recordCommunityEndpoint, updateRecordCallback } = this.props;
+    this.setState({ loading: true });
+    try {
+      const response = await http.put(recordCommunityEndpoint, {
+        data: value,
+      });
+
+      updateRecordCallback(response.data.result);
+      this.setState({ loading: false });
```

Comment:
maybe this should be moved before the updateRecordCallback as the latter will trigger a rerender of the ManageDefaultBrandingAction component with the wrong loading value momentarily!